src/app/core/breadcrumbs/breadcrumbs.component.ts

```diff
@@ -1,11 +1,29 @@
-import { Component } from '@angular/core';
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { Subscription } from 'rxjs';
+
+import { BreadcrumbsService } from '../../shared/services/breadcrumbs/breadcrumbs.service';
 
 @Component({
   selector: 'app-breadcrumbs',
   templateUrl: './breadcrumbs.component.html',
   styleUrls: ['./breadcrumbs.component.sass']
 })
-export class BreadcrumbsComponent {
-  public crumbsArray = ['Courses'];
-  public currentPage = 'Courses';
+export class BreadcrumbsComponent implements OnInit, OnDestroy {
+  public crumbsArray: string[];
+  public currentPage: string;
+  private sub: Subscription;
+
+  constructor(private breadcrumbsService: BreadcrumbsService) {}
+
+  ngOnInit() {
+    this.sub = this.breadcrumbsService.getBreadcrumbs().subscribe(data => {
+      this.crumbsArray = data;
+      this.currentPage = this.crumbsArray[this.crumbsArray.length - 1];
+    });
```

Comment:
I'd refactor it with to use async pipe. `crumbsArray$` and `crumbsArray$`, for example.